python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -244,7 +370,7 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
                 </React.Fragment>
               )}{" "}
               (Profile for
-              {[10, 30, 60].map((duration) => (
+              {[10, 30, 60].map(duration => (
                 <React.Fragment>
```

Comment:
I think you had added a `key` here before merging `master`, can we add that back?